packages/adobe/adobe-analytics/src/index.ts

```diff
@@ -1,2 +1,2 @@
 export { AaAdapter } from './adapter';
-export { integration as default } from './integration';
+export { integration } from './integration';
```

Comment:
I think the `default` will be necessary downstream.
```suggestion
export { integration, integration as default } from './integration';
```